src/Dodo.HttpClient.ResiliencePolicies/HttpClientBuilderExtensions.cs

```diff
@@ -30,13 +30,20 @@ public static class HttpClientBuilderExtensions
 			string clientName = null) where TClientInterface : class
 			where TClientImplementation : class, TClientInterface
 		{
-			var httpClientBuilder = sc.AddHttpClient<TClientInterface, TClientImplementation>(client =>
-				{
-					client.BaseAddress = baseAddress;
-					client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-					client.Timeout = settings.HttpClientTimeout;
-				})
-				.AddDefaultPolicies(settings);
+			Action<System.Net.Http.HttpClient> defaultClient = (client) => 
+			{
+				client.BaseAddress = baseAddress;
+				client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+				client.Timeout = settings.HttpClientTimeout;
+			};
+
+			IHttpClientBuilder httpClientBuilder;
+			if (!string.IsNullOrEmpty(clientName))
+				httpClientBuilder = sc.AddHttpClient<TClientInterface, TClientImplementation>(clientName, defaultClient);
```

Comment:
Please, could you add curly brackets for the single-line if-statements. I know, there are lack of style guide for this, but we are usually prefer to use it for any if-statement body for consistency.

Also, I ask you to add it to check my changes to the PR-check workflow. I will merge your PR anyway, but lets give it one more try.

Reply:
did it.